src/js/providers/youtube.js

```diff
@@ -247,6 +247,10 @@ define([
             switch (_youtubeState) {
 
                 case youtubeStates.UNSTARTED: // -1: //unstarted
+                    // play video on android to avoid being stuck in this state
+                    if (utils.isAndroid()) {
+                        _youtubePlayer.playVideo();
```

Comment:
How are you sure we want to play when the YT player state changes to unstarted or cued? Couldn't YT enter this state during our player's complete, idle or preload phase (no autostart)?